homeassistant/components/solarlog/sensor.py

```diff
@@ -46,55 +46,28 @@ async def async_setup_entry(hass, entry, async_add_entities):
     data = await hass.async_add_executor_job(SolarlogData, hass, api, host)
 
     # Create a new sensor for each sensor type.
-    entities = []
-    for sensor_key in SENSOR_TYPES:
-        sensor = SolarlogSensor(entry.entry_id, device_name, sensor_key, data)
-        entities.append(sensor)
-
+    entities = [
+        SolarlogSensor(entry.entry_id, device_name, data, description)
+        for description in SENSOR_TYPES
+    ]
     async_add_entities(entities, True)
     return True
 
 
 class SolarlogSensor(SensorEntity):
     """Representation of a Sensor."""
 
-    def __init__(self, entry_id, device_name, sensor_key, data):
+    def __init__(
+        self, entry_id, device_name, description: SolarLogSensorEntityDescription, data
```

Comment:
```suggestion
        self, entry_id, device_name, data, description: SolarLogSensorEntityDescription,
```
The arguments don't match the call above. Probably happened with the last change.